build.xml

```diff
@@ -688,6 +688,7 @@
           </dependency>
           <dependency groupId="org.hamcrest" artifactId="hamcrest" version="2.2" scope="test"/>
           <dependency groupId="com.github.seancfoley" artifactId="ipaddress" version="5.3.3" />
+          <dependency groupId="org.agrona" artifactId="agrona" version="0.9.26" />
```

Comment:
I think we will need a thread on the dev list to get consensus about adding the new dependency. I guess it won't be a problem.

Reply:
Started discussion thread.